src/views/shared/workflow-history-event-details-wf-execution-link/workflow-history-event-details-wf-execution-link.tsx

```diff
@@ -11,14 +11,20 @@ export default function WorkflowHistoryEventDetailsExecutionLink({
   cluster,
   domain,
 }: Props) {
-  const noMissingData = Boolean(domain && cluster && workflowId && runId);
+  const linkText = runId || workflowId;
```

Comment:
Is it possible for the props to have runId but not workflowId? If not, I think you can rewrite this code to avoid the workflowId check in Line 18

Reply:
I think you could do something like this to make the two cases clearer:

```
if (!domain || !cluster || !workflowId) return null;

let linkText = '', href= '';
if (runId) {
    // text, link to wf run
} else {
    // text, link to wf search page
}
```